doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -847,6 +833,43 @@ Modem libraries
       Refer to the :ref:`migration guide <migration_2.8>` for more details.
     * The :c:enum:`lte_lc_reduced_mobility_mode` type has been deprecated.
       Refer to the :ref:`migration guide <migration_2.8>` for more details.
+    * The library to be reorganized into modules that are enabled via their respective Kconfig options.
+      By default, the library now enables only the core features related to the network connectivity.
+      This reorganization reduces flash memory consumption for applications that only use the core features of the library related to network connectivity.
+      For more information on how to update your project, see the :ref:`migration guide <migration_2.8_required>`.
```

Comment:
```suggestion
    * The library was reorganized into modules that are enabled through their respective Kconfig options.
      By default, the library now enables only the core features related to the network connectivity.
      This reorganization reduces flash memory consumption for applications that only use the core features of the library.
      For more information on how to update your project, see the :ref:`migration guide <migration_2.8_required>`.
```

Reply:
I fully agree it clashes with the convention, but this still reads a bit better. Let's not get stuck with this. There will still be a round for mixed doc improvements and my team will figure out if we can make this better.